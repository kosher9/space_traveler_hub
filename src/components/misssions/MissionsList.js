import Mission from './Mission';

const MissionsList = () => (
  <table className="border border-slate-400 w-11/12 mx-auto my-5">
    <tbody className="border border-slate-400">
      <tr className="border border-slate-400">
        <th className="border border-slate-400">Mission</th>
        <th className="border border-slate-400">Description</th>
        <th className="border border-slate-400">Status</th>
        <th className="border border-slate-400"> </th>
      </tr>
      <Mission />
    </tbody>
  </table>
);

export default MissionsList;
