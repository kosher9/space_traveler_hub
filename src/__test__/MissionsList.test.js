import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../redux/configureStore';
import MissionsList from '../components/misssions/MissionsList';

describe('Missions tests', () => {
  it('render test', () => {
    const tree = renderer.create(
      <Provider store={store}>
        <MissionsList />
      </Provider>,
    );

    expect(tree).toMatchSnapshot();
  });
});
