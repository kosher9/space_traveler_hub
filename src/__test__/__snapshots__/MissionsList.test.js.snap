// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Missions tests render test 1`] = `
<table
  className="border border-slate-400 w-11/12 mx-auto my-5"
>
  <tbody
    className="border border-slate-400"
  >
    <tr
      className="border border-slate-400"
    >
      <th
        className="border border-slate-400"
      >
        Mission
      </th>
      <th
        className="border border-slate-400"
      >
        Description
      </th>
      <th
        className="border border-slate-400"
      >
        Status
      </th>
      <th
        className="border border-slate-400"
      >
         
      </th>
    </tr>
  </tbody>
</table>
`;
